src/lib.rs

```diff
@@ -8,10 +15,11 @@ pub mod prototype;
 mod utils;
 
 pub mod prelude {
-    pub use bevy_proto_derive::*;
+    //! Includes all public types and the macro to derive [`ProtoComponent`](super::components::ProtoComponent).
```

Comment:
Is it worth suggesting that it's often best used as `use bevy_proto::prelude::*`? Or is it better to just let the user decide how they want to use it?

Reply:
I think this might depend on how they're using it, and if they're using all of the `bevy_proto` types in the same place. I'd just let users decide how to use it.

On a related note I was thinking we could reduce the number of public modules. Specifically, we could hide `components` and `plugin` and just `pub use components::ProtoComponent` and `pub use plugin::ProtoPlugin` since those mods don't contain anything else anyway.

Reply:
Yeah, let's do that.